DEVELOPMENT.md

```diff
@@ -107,3 +84,82 @@ Now you can use your new SDK and call the following methods with various params
 * `Hosts.totals()`
 * `Hosts.get()`
 * `Hosts.delete()`
+
+### Tests
+
+This project contains:
+- [Datadog API Client](/datadog/api)
+- [Dogshell](/datadog/dogshell)
+- [DogStatsD](/datadog/dogstatsd)
+- [Threadstats](/datadog/threadstats)
+
+
+We have [unit](/tests/unit), [integration](/tests/integration) and [performance](/tests/performamce) tests.
+Integration tests need an _API_ and _APP Keys_ to run against a Datadog account.
+- __WARNING__: Never use keys for an organization that contains anything important.
+
+We use `tox` to run tests. You can find the [tox.ini](/tox.ini) config in the root directory.
+We create 2 environments:
```

Comment:
technically, there are 10 environments:  
- the 9 that are run when invoking simply `tox`. Every pyXX env runs unit tests:
```
❯ tox -l
py27
py35
py36
py37
py38
pypy2
pypy3
flake8
integration
```
- the `integration-admin` environment that you have to explicitly call: `tox -e integration-admin`